messageapp/src/clients/createBudget.js

```diff
@@ -0,0 +1,25 @@
+import Budget from '../models/budget.js'
+
+export default async (budgetParams) => {
+
+    const checkIfBudget = await Budget.find()
+
+    if (checkIfBudget.length != 0) {
+
+        console.log("Budget already created")
+
+    } else {
+
+        const budget = new Budget(budgetParams);
+
+        try {
```

Comment:
Maybe you need to lock the database, just in case you need to do a rollback (hint: try to use a lock library)